wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java

```diff
@@ -44,19 +41,17 @@ public abstract class RobotBase implements AutoCloseable {
 
   private final int m_connListenerHandle;
 
-  private boolean m_dashboardDetected;
-
   private static void setupCameraServerShared() {
     CameraServerShared shared =
         new CameraServerShared() {
           @Override
           public void reportVideoServer(int id) {
-            HAL.report(tResourceType.kResourceType_PCVideoServer, id + 1);
+            HAL.reportUsageCount("CameraServer", id + 1);
```

Comment:
Just realized that I missed that this part also still adds 1 to the ID. (Then again, C++ also didn't add 1 before, so maybe this is actually a language difference?)

Reply:
This code is going away when I change CameraServerShared to use just a single report function like what I did with MathShared.